indico/modules/events/abstracts/settings.py

```diff
@@ -53,6 +71,14 @@ class BOACorrespondingAuthorType(RichEnum):
 }
 
 
+BOALinkFormatting.__titles__ = {
+    BOALinkFormatting.frame: _('Frame'),
+    BOALinkFormatting.colorlinks: _('Color Links'),
+    BOALinkFormatting.ocgcolorlinks: _('Color Links for screen only'),
+    BOALinkFormatting.ocgx2colorlinks: _('Color Links for screen only(with ogcx2)'),
```

Comment:
What's the difference to the option above? A typical user will not know what "ogcx2" is so it's a bit confusing. (also, missing space before the `(`).